inc/learndash.php

```diff
@@ -35,3 +35,21 @@ function ( $content, $post ) {
 	10,
 	2
 );
+
+remove_filter( 'learndash_template', 'learndash_course_grid_course_list', 999999, 5 );
```

Comment:
can this be put inside a check to see if learndash_course_grid plugin is present and active please?